.github/workflows/backend-cli-test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        etcd: [3.4.13]
+        etcd: [3.4.14]
```

Comment:
we could update to 3.4.16

Reply:
there is no image of etcd:3.4.16, so just keep 3.4.14 here